src/api/localResolvers/activeLoan.js

```diff
@@ -0,0 +1,65 @@
+/*
+ * Active loan resolvers
+ */
+export default () => {
+	return {
+		defaults: {
+			activeLoan: {
+				hoverLoanId: 0,
+				xCoordinate: 0,
+				yCoordinate: 0,
+				loan: JSON.stringify({
+					activity: {},
+					userProperties: {},
+					loanFundraisingInfo: {},
+					geocode: {
+						country: {},
+					},
+					image: {},
+				}),
+				tracking: JSON.stringify({}),
+				__typename: 'ActiveLoan',
+			}
+		},
+		resolvers: {
+			Mutation: {
+				updateActiveLoan(
+					_,
+					{
+						hoverLoanId = 0,
+						xCoordinate = 0,
+						yCoordinate = 0,
+						loan = JSON.stringify({
+							activity: {},
+							userProperties: {},
+							loanFundraisingInfo: {},
+							geocode: {
+								country: {},
+							},
+							image: {},
+						}),
+						tracking = JSON.stringify({}),
```

Comment:
@emuvente Any thoughts on this approach. I'm guessing that by simplifying the data type to a string it just makes it easier to deal with adding a complex data type here.

Reply:
I think it's fine for the tracking (maybe have the name indicate it though, like trackingJson or trackingBlob). For the loan though I think it might be better to have it be an actual loan type or maybe just a loan id. That should allow taking advantage of the cache normalization and will not duplicate the loan data in memory.

Reply:
I agree encoding as a string is definitely suboptimal. In our next revision we will update the way we do this.